ecosystem/sep-0012.md

```diff
@@ -90,16 +90,16 @@ Name | Type | Description
 `account` | `G...` string | (optional) The Stellar account ID used to identify this customer. If many customers share the same Stellar account, the `memo` and `memo_type` parameters should be included as well.
 `memo` | string | (optional) a [properly formatted memo](https://developers.stellar.org/docs/glossary/transactions/#memo) that uniquely identifies a customer. This value is generated by the client making the request. This parameter and `memo_type` are identical to the `PUT` request parameters of the same name.
 `memo_type` | string | (optional) type of `memo`. One of `text`, `id` or `hash`. If `hash`, `memo` should be base64-encoded.
-`type` | string | (optional) the type of action the customer is being KYCd for.  See below.
+`type` | string | (optional) the type of action the customer is being KYCd for.  See the [Type Specification](#type-specification) below.
 `lang` | string | (optional) Defaults to `en`.  Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).  Human readable descriptions, choices, and messages should be in this language.
 
 #### ID vs. Account & Memo
 
 The client can always use the `account`, `memo`, and `memo_type` parameters to uniquely identify a customer. However, if a `PUT /customer` request has already been made for the customer, the client may use the `id` returned in the response instead.
 
-#### Type specification
+#### Type Specification
 
-Different types of actions may have different kyc requirements.  The type parameter is used to signify what type of transaction this customer needs to be KYC'd for.  For example, if a customer is being KYC'd as a SEP31 sender, they may only require full name, but a SEP31 receiver needs to be fully KYCd with comprehensive data.  It can also be used to specify if this is a business being KYC'd or a person which can require different fields.  The value used for `type` is up to the implementor to signify the types of transactions their services support, and is optional.
+Different types of customers may have different KYC requirements depending on the action that customer wants to take. The type parameter is used to specify the action a customer wants to make so the server can identify what information must be collected. For example, if a customer is being KYC'd as a SEP31 sender, they may only require full name and email, but a SEP31 receiver would require banking information. The `type` parameter could also be used to specify if an organization or person is being KYC'd, which would require different fields. The values accepted for `type` are up to the implementor in order to signify the types of transactions their services support, and is optional if only one type of customer is needed.
```

Comment:
I think we need to explicitly state that the values of `type` are undefined by SEP-12 and may be defined by SEPs that utilize SEP-12.

Reply:
Done. I took out the part where I said `the values accepted for type are up to the implementor` because that information can be communicated by the protocol defining the types.